package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "action-docs": "git+https://github.com/deemp/action-docs.git#0cb0ac2c40736aba24c4fd5097f1074b5a7c7a4b"
```

Comment:
:exploding_head:

Reply:
Well, `npm update action-docs` works, the SHA can be removed.